app/src/main/java/org/jellyfin/mobile/settings/SettingsFragment.kt

```diff
@@ -90,10 +86,23 @@ class SettingsFragment : Fragment() {
             titleRes = R.string.external_player_app
             enabled = appPreferences.videoPlayerType == VideoPlayerType.EXTERNAL_PLAYER
         }
+        categoryHeader(PREF_CATEGORY_DOWNLOADS) {
+            titleRes = R.string.pref_category_downloads
+        }
+        val downloadsDirs: List<SelectionItem> = mutableListOf<SelectionItem>().apply {
+            requireContext().getDownloadsPaths().forEach {
+                add(SelectionItem(it, it, null))
+            }
+        }
+        singleChoice(Constants.PREF_DOWNLOAD_LOCATION, downloadsDirs) {
+            titleRes = R.string.pref_download_location
+            initialSelection = appPreferences.downloadLocation
+        }
```

Comment:
I suppose you could move the download option to the top (and call the category something like "General")